assets/RandomComicViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  RandomComicViewController.swift
+//  xkcd
+//
+//  Created by Ekam Singh Dhaliwal on 2019-09-04.
+//
+
+import UIKit
+
+class RandomComicViewController: UIViewController {
+
+  @IBOutlet weak var comicImage: UIImageView!
+
+
+  override func viewDidLoad() {
+        super.viewDidLoad()
+
+
+    }
+
+
+  @IBAction func generateComic(_ sender: UIButton) {
+
+    fetchComicData(completion: displayImage(comic:))
+
+  }
+
+
+
+
+
+  func generateRandomNumber() -> Int {
+    return Int.random(in: 0 ... 2198)
+  }
```

Comment:
I think I missed a user story... When the app starts us we likely want it to load the most recent comic. At that point, we would know what the most recent comic id is & we could use that as the upper bound in this generateRandomNumber() function instead of hard coding 2198. A new comic comes out every week. 

Can you create a new issue for this user story and make a note to update this hard coded number in generateRandomNumber when it's implemented.